elements/urth-core-channels/urth-core-channels.html

```diff
@@ -127,7 +129,11 @@
          */
         _handleItemChangeMessage: function( path, value ){
             var path = path.split(':');
-            this.setItem(path[1], value, path[0]);
+            if (!this._getChannel(path[0])){
```

Comment:
Aren't we doing getOrCreate for channels? We probably will never hit this condition of the channel not existing.